scripts/rpm-upgrade.sh

```diff
@@ -148,10 +148,16 @@ bb_log_info "Package_list: $package_list"
 # sudo rm -rf
 # /etc/zypp/repos.d/SUSE_Linux_Enterprise_Server_12_SP3_x86_64:SLES12-SP3-Updates.repo
 # /etc/zypp/repos.d/SUSE_Linux_Enterprise_Server_12_SP3_x86_64:SLES12-SP3-Pool.repo
-sudo "$pkg_cmd" clean all
+if [[ $ID_LIKE =~ ^suse* ]]; then
+  sudo "$pkg_cmd" clean --all
+  pkg_cmd_nonint="-n"
+else
+  sudo "$pkg_cmd" clean all
+  pkg_cmd_nonint="-y"
```

Comment:
Good, but I would change the naming of the variable in case we need other options in the future, something like `pkg_cmd_options`.

Reply:
Changed, thanks!